### PR TITLE
Add and use self-hosted fonts

### LINKED ISSUE #123: Use self hosted Google Fonts
Use self hosted Google Fonts instead of loaded over Google API.

Improve compatibility to [GDPR](https://cookie-script.com/blog/google-fonts-and-gdpr).

### PR DESCRIPTION
**Pull request by SSirjes**
Add and use self hosted fonts to improve GDPR compliance.
Closes https://github.com/Voog/design-dusseldorf/issues/123

### CODE REVIEW

**Review by t-p-developer — COMMENTED** [review]
Looks okay, but I don't understand why package-lock.json was included on this commit ...

File: sources/stylesheets/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,504 @@
+@font-face {
+    font-family: 'Anonymous Pro';
+    src: url('../assets/anonymous-pro-v21-latin-ext_greek_cyrillic-regular.woff') format('woff');
+    font-weight: normal;
+  }
```
Comment (PeeterPaal) [inline sources/stylesheets/_fonts.scss]: Indentation seems to be off here and below, which would be nice to correct.

**Review by PeeterPaal — APPROVED** [review]
Looks good! Just one small comment.

**Review by tanelj — APPROVED** [review]
Nice!

**Pull request merged**